app/api/documents.py

```
@@ -0,0 +1,90 @@
+            )
+            embedded_chunks.append({
+                "content": chunk.content,
+                "embedding": embeddings[0],
+                "metadata": {
+                    "doc_name": request.metadata.doc_name,
+                    "page": chunk.page,
+                    "category": request.metadata.category,
+                    "enforcement_date": str(request.metadata.enforcement_date),
+                },
+            })
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)